.werft/jobs/build/self-hosted-upgrade-tests.ts

```diff
@@ -46,6 +46,11 @@ export async function triggerUpgradeTests(werft: Werft, config: JobConfig, usern
     exec(`git config --global user.name "${username}"`);
     var annotation = `-a version=${config.fromVersion} -a upgrade=true -a channel=${channel} -a preview=true -a skipTests=true`;
 
+    // the following bit make sure that the subdomain stays the same upon multiple runs, and always start with release
+    const regex = /[\/,\.]/g;
+    const subdomain: string = config.repository.branch.replace(regex, '-')
+    annotation = annotation.concat(`-a subdomain=${subdomain}`)
```

Comment:
```suggestion
    annotation = annotation.concat(` -a subdomain=${subdomain}`)
```

I think this will set the `annotation` as `-a skipTests=true-a subdomain=${subdomain}`